TerminalDocs/group-policy.md

```diff
@@ -0,0 +1,68 @@
+---
+title: Group Policy
+description: Learn how to set Group Policy for Windows Terminal.
+author: chrnguyen
+ms.author: chrnguyen
+ms.date: 10/24/2024
+ms.topic: how-to 
+---
+
+# Group Policies
+
+Since Windows Terminal Preview 1.22, Windows Terminal is released on GitHub with Administrative Templates that allow you to configure Windows Terminal using Group Policies.
+
+## How to install
+
+### Download
+
+You can find the latest administrative templates (ADMX and ADML files) in the assets section of our newest Windows Terminal release on GitHub. The file is named `GroupPolicyTemplates-<Version>.zip`
+
+### Add the administrative template to an individual computer
+
+1. Copy the `WindowsTerminal.admx` file to your **Policy Definition** template folder. (Example: `C:\Windows\PolicyDefinitions`)
+2. Copy the `WindowsTerminal.adml` file to the matching language folder in your language folder in your Policy Definition folder. (Example: `C:\Windows\PolicyDefinitions\en-US`)
```

Comment:
Should we simplify the two steps to just say that one should copy the .zip contants as-is into the policy folder?

Reply:
If it contains admx and en-us folder with adml, extracting the zip is as simple as possible.

Reply:
Good points. I tried to follow the patterns set in the [PowerToys Group Policy doc](https://learn.microsoft.com/en-us/windows/powertoys/grouppolicy#add-the-administrative-template-to-an-individual-computer) when I wrote this. 
Since we only have en-us right now, perhaps having one step that says extracting the zip to Policy Definition is the best option :)